src/ert/config/forward_model_step.py

```diff
@@ -18,6 +25,25 @@
 logger = logging.getLogger(__name__)
 
 
+class ForwardModelStepJSON(TypedDict):
```

Comment:
Since this becomes part of the public API, we probably need docstrings wherever we can have it in this file.